src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -130,7 +131,8 @@ private bool TryMoveKeyboardFocusAndSelection(int offset, SelectorItem focusedCo
 			}
 
 			// If selection mode is single, moving focus also selects the item
-			if (SelectionMode == ListViewSelectionMode.Single)
+			if (SelectionMode == ListViewSelectionMode.Single &&
```

Comment:
Are we able to test this new feature with automation peers?

Reply:
I think it is not possible, as the only way to get into this portion of code is from `OnKeyDown`, and programmatic focus does not seem to move selection this way even if `SingleSelectionFollowsFocus` on UWP/WinUI

Reply:
Got it. Let's mark the UI Test as a manual test then.

Reply:
@jeromelaban Done!